cluster-autoscaler/cloudprovider/huaweicloud/README.md

```diff
@@ -0,0 +1,229 @@
+# Cluster Autoscaler on Huawei Cloud 
+
+## Overview
+The cluster autoscaler for [Huawei Cloud](https://www.huaweicloud.com/) scales worker nodes within any
+specified Huawei Cloud Container Engine (CCE) cluster's node pool where the `Autoscaler` label is on. 
+It runs as a Deployment on a worker node in the cluster. This README will go over some of the necessary steps required 
+to get the cluster autoscaler up and running.
+
+Note: 
+
+1. Cluster autoscaler must be run on CCE v1.15.6 (Kubernetes v1.15) or later.
+2. Node pool attached to the CCE cluster must have the `Autoscaler` flag turned on, and minimum number of nodes and maximum
+number of nodes being set. Node pools can be managed under `Resource Management` in CCE console.
+3. If warnings about installing `autoscaler addon` are encountered after creating a node pool with `Autoscaler` flag on, 
+just ignore this warning and DO NOT install the addon.
+4. Do not build your image in a Huawei Cloud ECS. Build the image in a machine that has access to the Google Container Registry (GCR).
+
+## Deployment Steps
+### Build Image
+#### Environment
+1. Download Project
+
+    Get the latest `autoscaler` project and download it to `${GOPATH}/src/k8s.io`. 
+    
+    This is used for building your image, so the machine you use here should be able to access GCR. Do not use a Huawei 
+    Cloud ECS.
+
+2. Go environment
+
+    Make sure you have Go installed in the above machine.
+    
+3. Docker environment
+
+    Make sure you have Docker installed in the above machine.
+    
+#### Build and push the image
+Execute the following commands in the directory of `autoscaler/cluster-autoscaler` of the autoscaler project downloaded previously.
```

Comment:
Cluster-autoscaler has its development flow. Do you need to list this separately? If anything is missing, could you contribute back to `cluster-autoscaler` root level?

Reply:
Thank you for the comment. Yeah you're right about the repetition of the development flow part. The pieces you quoted above have already been mentioned in the root level of cluster autoscaler. The reason I consider it may still be necessary is that the repetition could make the steps of deploying CA on Huawei Cloud CCE more complete, so that people who get this code can follow the **exact steps** described in the README.md file to make the CA working on CCE.

Please let me know if you still think it unnecessary so I can get rid of that part.